package.json

```diff
@@ -24,6 +24,7 @@
     "test": "asp && npm run test:ts",
     "test:ci": "asp --summary --config as-pect-ci.config.js",
     "test:proba": "asp --summary --config as-pect-proba.config.js",
+    "test:types": "asp --summary --config as-pect-types.config.js",
```

Comment:
forgot to commit `as-pect-types.config.js`

Reply:
Done